crates/bevy_asset/src/loader.rs

```diff
@@ -20,6 +20,11 @@ use thiserror::Error;
 
 /// Loads an [`Asset`] from a given byte [`Reader`]. This can accept [`AssetLoader::Settings`], which configure how the [`Asset`]
 /// should be loaded.
+///
+/// This trait defines asset-specific logic to convert serialized data into Rust types.
+/// For asset-agnostic mechanisms to interact with a storage system, see [`AssetSaver`](crate::saver::AssetSaver) and [`AssetLoader`](crate::loader::AssetLoader).
```

Comment:
This line feels redundant, or at the very least confusing. Why is it recommending that same type it is documenting as tho it were an alternative, or something different, to the documented type? Was this meant to be a different type being recommended here?

(Ditto for the duplicate line on `AssetSaver` which is the same, with the same questions I mentioned here).

Reply:
Yeah I agree that this is a bit awkward. It feels phrased as if it was describing AssetReader and AssetWriter.